clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1417,6 +1417,7 @@ private void recordDiskQuotaAllowedForStore(Store store) {
     if (emitMetrics.get()) {
```

Comment:
Minor: we might just change the method name to `recordDiskQuotaMetricsForStore`.